.github/workflows/support-window-issue.yml

```diff
@@ -0,0 +1,36 @@
+name: Support Windows Update Issue Creater
+on:
+  pull_request:
```

Comment:
we can remove this trigger now

Reply:
Removed the triggers which were added for testing.